package.json

```diff
@@ -33,17 +33,18 @@
     "lint": "eslint src"
   },
   "dependencies": {
+    "@babel/helper-module-imports": "7.0.0-beta.31",
     "rollup-pluginutils": "^1.5.0"
   },
   "peerDependencies": {
     "@babel/core": "7 || ^7.0.0-beta"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.5",
-    "@babel/plugin-external-helpers": "^7.0.0-beta.5",
-    "@babel/plugin-proposal-decorators": "^7.0.0-beta.5",
-    "@babel/plugin-transform-runtime": "^7.0.0-beta.5",
-    "@babel/preset-env": "^7.0.0-beta.5",
+    "@babel/core": "7.0.0-beta.31",
```

Comment:
If we are already updating maybe use .35?

Reply:
sure thing, it's just already quite old PR and .31 was the latest back then 😄